settings.gradle

```diff
@@ -19,4 +22,28 @@ buildCache {
 		enabled = true
 		removeUnusedEntriesAfterDays = 7
 	}
+	Properties localProperties = new Properties()
+	File file = new File("local.properties")
+	if (file.exists()) {
+		localProperties.load(file.newDataInputStream())
```

Comment:
Our publishing script shows how to load properties and close the file input stream afterwards to prevent the leak https://github.com/GetStream/stream-chat-android/blob/main/scripts/publish-root.gradle#L13